news/1355.feature

```diff
@@ -0,0 +1 @@
+Add portal_type title (`type`) to content response [razvanMiu]
```

Comment:
```suggestion
Add portal_type title (`type_name`) to content response [razvanMiu]
```

Reply:
(Why do we call it name instead of title, since it's the title?)

Reply:
@davisagli type confuses with `@type` and changed as per https://github.com/plone/plone.restapi/pull/1355#discussion_r862338290